pages/linux/conky.md

```diff
@@ -0,0 +1,27 @@
+# conky
+
+> Light-weight system monitor for X.
+
+- Launch with default, built-in config:
+
+`conky`
+
+- Create a new, default config:
```

Comment:
I don't think the comma is necessary here. :thinking:

Reply:
Yeah I agree, it's a [cumulative adjective](https://www.quickanddirtytips.com/education/grammar/commas-with-adjectives) :smile: